Updated troubleshooting section (#16)
ReVanced-Extended-Community/Community-Guides: general-guides/community-wiki/ytm-troubleshooting.md

### **1. No internet connection / The home page isn't loading**

Navigate to your device settings > Accounts > Manage accounts > GmsCore > Remove account. You can sign in again if you want.

> > If that doesn't work make sure you have the latest version of [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest) installed.

Reviewer: ```suggestion
> If that doesn't work you can uninstall GmsCore and then reinstall [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).

```

### **2. I'm having issues signing in to my Google account**

Navigate to your device settings > Accounts > Manage accounts > GmsCore > Remove account. You can sign in again if you want.

> > If that doesn't work make sure you have the latest version of [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest) i

Reviewer: Refer to my comment on issue 2 in the yt troubleshooting page.

### **3. The download feature isn't working**

> Refer to step 2b of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/wiki-redesign/general-guides/community-wiki/patching%20%26%20setup%20info.md#2b-downloader-setup) to understand and set up the download function in YTM Extended.

Reviewer: ```suggestion
Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YTM Extended.

```

### **4. How do I get YTM Extended on Android Auto?**

> Refer to step 3a of the [setup guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3aandroid-auto-setup).

Reviewer: ```suggestion
Refer to section 3 of the [setup guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3-yt-music-setup).

```

### **5. Why can I only use the Radio feature?**

Google has made it so that Canadian users who do not have YT Premium can only use the Radio feature within YouTube Music.

> Refer to step 3b of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3b-spoof-app-version-setup-for-canadian-users) to solve this.

Reviewer: ```suggestion
Refer to section 3 of the [setup guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3-yt-music-setup) to solve this.

```

### **1. No internet connection / Home page not loading / Account sign-in issues**

Navigate to your device settings > Accounts > Manage accounts > Google (blue icon) > Remove account. You can sign in again if you want.

> If that doesn't work you can uninstall GmsCore and then reinstall [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).

### **2. The download feature isn't working**

> Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YTM Extended.

Reviewer: ```suggestion
Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YT Music RVX.

```

> ### **3. How do I get YTM Extended on Android Auto?**

Reviewer: ```suggestion
### **3. How do I get YT Music RVX on Android Auto?**

```

Author: Is the app name really YT Music RVX? Bit convoluted huh?

Google has made it so that Canadian users who do not have YT Premium can only use the Radio feature within YouTube Music.

> Refer to step 3b of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3b-spoof-app-version-setup-for-canadian-users) to solve this.

Reviewer: ```suggestion
Refer to section 3 of the [setup guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3-yt-music-setup) to solve this.

```
